controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java

```diff
@@ -122,68 +99,42 @@ public PathElement getPathElement() {
 
     public void parse(final XMLExtendedStreamReader reader, PathAddress parentAddress, List<ModelNode> list) throws XMLStreamException {
         ModelNode op = Util.createAddOperation();
+
         boolean wildcard = resourceDefinition.getPathElement().isWildcard();
-        String name = parseAttributeGroups(reader, op, wildcard);
+        String name = parseRootAttributes(reader, op, wildcard);
         if (wildcard && name == null) {
             throw ControllerLogger.ROOT_LOGGER.missingRequiredAttributes(new StringBuilder(NAME), reader.getLocation());
         }
+
         PathElement path = wildcard ? PathElement.pathElement(resourceDefinition.getPathElement().getKey(), name) : resourceDefinition.getPathElement();
         PathAddress address = parentAddress.append(path);
-        if (!noAddOperation) {
+        if(!noAddOperation) {
```

Comment:
Please adjust your IDE to not make whitespace changes to lines otherwise unedited. Also, removing the space is incorrect.